custom_components/tahoma/__init__.py

```diff
@@ -222,11 +222,11 @@ def log_device(message: str, device: Device) -> None:
 async def _block_if_core_is_configured(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     overkiz_config_entries = hass.config_entries.async_entries("overkiz")
 
-    for overkiz_entry in overkiz_config_entries:
-        if (
-            entry.data[CONF_USERNAME] == overkiz_entry.data[CONF_USERNAME]
+    return any(
+        (
+            overkiz_entry.source != "ignore"
```

Comment:
Most of the time, there is always a constant, like [here](https://github.com/home-assistant/core/blob/494ef2f9b2e5feeb10dce94a02ee90a9a7430b6a/homeassistant/config_entries.py#L54)

Reply:
can be several things depending on how the user added the integration. For us, I guess one of those:
```py
SOURCE_USER = "user"
SOURCE_ZEROCONF = "zeroconf"
SOURCE_DHCP = "dhcp"
```